dashboard/app/views/lti/v1/authenticate/unsupported_message_type.html.haml

```diff
@@ -0,0 +1,12 @@
+#lti-unsupported-message-type.error-page
+  .error-parent
+    .error-child
+      = image_tag '/shared/images/sad-bee-avatar.png', alt: t('lti.error.unsupported_message_type')
```

Comment:
I would use an empty alt text for the sad bee since:

1.  The image does NOT contain text.
2. The image is NOT used in a link or a button.
3. The image does NOT contribute meaning to the current page or context.
4. The alt text is used in the title below.

Given those cases, I would consider the image [decorative](https://www.w3.org/WAI/tutorials/images/decorative/).